environments/servlet/core/src/main/java/org/jboss/weld/environment/jetty/JettyContainer.java

```diff
@@ -71,14 +67,8 @@ public boolean touch(ContainerContext context) throws Exception {
     public void initialize(ContainerContext context) {
         // Try pushing a Jetty Injector into the servlet context
         try {
-            Class<?> clazz = Reflections.classForName(JettyWeldInjector.class.getName());
-            Object injector = clazz.getConstructor(WeldManager.class).newInstance(context.getManager());
-            context.getServletContext().setAttribute(INJECTOR_ATTRIBUTE_NAME, injector);
-
-            Class<?> decoratorClass = Reflections.classForName("org.jboss.weld.environment.jetty.WeldDecorator");
-            Method processMethod = decoratorClass.getMethod("process", ServletContext.class);
-            processMethod.invoke(null, context.getServletContext());
-
+            context.getServletContext().setAttribute(INJECTOR_ATTRIBUTE_NAME, new JettyWeldInjector(context.getManager()));
+            WeldDecorator.process(context.getServletContext());
```

Comment:
I don't think so. `JettyContainer.initialize(ContainerContext)` will not be called in a non-Jetty environment.